Source/UI/CommonUI/View/Component/InputView.swift

```diff
@@ -89,6 +89,10 @@ class InputView: UIView {
         updateTextFieldContainer(style: style)
         textFieldView.update(with: style.textfield)
         updateErrorView(style: style)
+
+        if let textFieldValue = textFieldValue {
+            textFieldView.textField.text = textFieldValue
+        }
```

Comment:
UI elements should not be overridden from another class. i suggest to set style.text
You may update it when update the style in  `CardHolderView` 
`func update(style: CellTextFieldStyle, text: String?)`
 ```suggestion
```

Reply:
Absolutely agree and to be honest this is why I would have liked the UI properties to always be private. Just thought if they're already internal, will use them...

Will change as I do agree with the point you raised ➕